scripts/install-mender.sh

```diff
@@ -202,7 +202,7 @@ get_deps() {
 add_repo() {
     curl -fsSL $REPO_URL/gpg | apt-key add -
 
-    repo="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
+    repo="deb [arch=$ARCH] $REPO_URL $DIST_VERSION/$CHANNEL main"
```

Comment:
Also, this is wrong. The codenames for our distros are:
* `debian/buster/experimental`
* `debian/buster/stable`
* `debian/bullseye/experimental`
* `debian/bullseye/stable`
* `ubuntu/bionic/experimental`
* `ubuntu/bionic/stable`
* `ubuntu/focal/experimental`
* `ubuntu/focal/stable`

So here we need to do something like:
```suggestion
    repo="deb [arch=$ARCH] $REPO_URL $LSB_DIST/$DIST_VERSION/$CHANNEL main"
```

And below we need to override `LSB_DIST` to `debian` when `raspbian` is detected.